### DISCUSSION TITLE
General Settings

### DISCUSSION THREAD

**Discussion by JB0710** [BackupNerd/Backup-Scripts#8] [General]
Is there a way to set the "Send To" email address under Preferences > General for the Backup dashboard settings and the Restore dashboard settings?

**Comment by JB0710** [comment]
Answered my own question:
$clienttool = "c:\program files\backup manager\clienttool.exe"
& "$clienttool" control.setting.modify -name MailAddress -value xxx@xxx.com

**Comment by BackupNerd** [comment]
There is.

And you found it.

Just Give yourself a 120-180 second wait before you try and set that value.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
